sycl/test/basic_tests/accessor/accessor_multi_ptr.cpp

```diff
@@ -0,0 +1,46 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
```suggestion
// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -fsyntax-only
```

See some BKMs on tests: [here](https://github.com/intel/llvm/blob/sycl/sycl/doc/developer/ContributeToDPCPP.md#dpc-headers-and-runtime-tests)

Since the test only contains a few `static_assert`s which do most of the actual checking, I would also suggest to maybe refactor the test to significantly simplify it. We can turn everything into a single function, which accepts `handler` and `buffer`, creates accessors and then checks return types of methods.

Reply:
thanks @AlexeySachkov, done.